<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>photoDesc</class>
 <widget class="QMainWindow" name="photoDesc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>photoDesc</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTreeView" name="fileTreeView">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>1</y>
      <width>256</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>421</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_button">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>360</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="toolTip">
     <string>Clear Description</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="edit-delete"/>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_comment_button">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>430</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Clear Comment</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="edit-delete"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_orientation">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>160</y>
      <width>75</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Exif Image.Orientation</string>
    </property>
    <property name="text">
     <string>orientation:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_oientation">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>160</y>
      <width>112</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>112</width>
      <height>0</height>
     </size>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_comment">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>410</y>
      <width>311</width>
      <height>81</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>440</y>
      <width>66</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Exif Photo.UserComment</string>
    </property>
    <property name="text">
     <string>Comment</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_title">
    <property name="geometry">
     <rect>
      <x>368</x>
      <y>356</y>
      <width>311</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>282</x>
      <y>356</y>
      <width>75</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Exif Image.ImageDescription</string>
    </property>
    <property name="text">
     <string>Description</string>
    </property>
   </widget>
   <widget class="QPushButton" name="restore_button">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>550</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="toolTip">
     <string>Restore orientation, description and comment to original values</string>
    </property>
    <property name="text">
     <string>Restore</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>550</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_datetaken">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>500</y>
      <width>254</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>0</height>
     </size>
    </property>
    <property name="toolTip">
     <string>Exif Photo.DateTimeOriginal</string>
    </property>
    <property name="text">
     <string>date taken</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_next">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>490</y>
      <width>30</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>30</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="go-next">
      <normaloff>.</normaloff>.</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_previous">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>530</y>
      <width>30</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>30</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="go-previous">
      <normaloff>.</normaloff>.</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_copy_desc">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>360</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Copy Description from previously displayed image</string>
    </property>
    <property name="text">
     <string>Copy Previous</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_copy_comment">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>430</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Copy Comment from previously displayed image</string>
    </property>
    <property name="text">
     <string>Copy Previous</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_undo_desc">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>360</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Undo Description Changes</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="edit-undo"/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_undo_comment">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>430</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Undo Comment Changes</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="edit-undo"/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>908</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
